Catchpoint/description.html

```diff
@@ -0,0 +1,83 @@
+<html>
+
+  <head>
+    <link rel="stylesheet" type="text/css" href="preview.css">
+  </head>
+
+  <body>
+    <!--<%def name="title()">Your Integration</%def>-->
+    <div class="sectiontitle" id="previewtitle"><h2>Catchpoint</h2></div>
+    <div class="saas_tile" id="tile">
+      <div class="saas_avatar saas_in">
+        <img src="Images/200x128.png">
+      </div>
+      <div class="saas_content">
+        <p class="saas_name saas_in">Catchpoint</p>
+        <div class="saas_controls">
+          <button class="meh">
+            <span class="js-icon text-success">+</span>
+            <span class="saas_msg" data-state_msg="Available" data-action_msg="Install">Available</span>
+          </button>
+        </div>
+        <div class="saas_description saas_in" id="tiledescription">Catchpoint is a Digital Performance Analytics platform that gives you the power to deliver amazing user experiences.</div>
+      </div>
+    </div>
+
+    <div class="integration_content" id="int-overview">
+      <h2>Overview</h2>
+      <p>Connect to Catchpoint to:</p>
+      <ul>
+        <li>Configure comprehensive alerts in your event stream</li>
+        <li>Direct links to Analysis Charts in the Catchpoint Portal</li>
+        <li>Alert Type tags for easy filtering</li>
```

Comment:
HI Ilan,

The integration only includes events at this point in time.

Cheers,

- Robert
> On 10 Feb 2016, at 16:17, Ilan Rabinovitch <notifications@github.com> wrote:
> 
> In Catchpoint/description.html <https://github.com/DataDog/IntegrationTemplate/pull/12#discussion_r52480802>:
> 
> > +          <button class="meh">
> > +            <span class="js-icon text-success">+</span>
> > +            <span class="saas_msg" data-state_msg="Available" data-action_msg="Install">Available</span>
> > +          </button>
> > +        </div>
> > +        <div class="saas_description saas_in" id="tiledescription">Catchpoint is a Digital Performance Analytics platform that gives you the power to deliver amazing user experiences.</div>
> > +      </div>
> > +    </div>
> > +
> > +    <div class="integration_content" id="int-overview">
> > +      <h2>Overview</h2>
> > +      <p>Connect to Catchpoint to:</p>
> > +      <ul>
> > +        <li>Configure comprehensive alerts in your event stream</li>
> > +        <li>Direct links to Analysis Charts in the Catchpoint Portal</li>
> > +        <li>Alert Type tags for easy filtering</li>
> Will the integration include any metrics, or just events?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/DataDog/IntegrationTemplate/pull/12/files#r52480802>.
>